htdocs/install/mysql/migration/15.0.0-16.0.0.sql

```diff
@@ -237,6 +237,8 @@ ALTER TABLE llx_advtargetemailing RENAME TO llx_mailing_advtarget;
 
 ALTER TABLE llx_mailing ADD UNIQUE uk_mailing(titre, entity);
 
+ALTER TABLE llx_inventory ADD COLUMN include_sub_warehouse boolean DEFAULT FALSE;
```

Comment:
Is it necessary to store the value ? Isn't it used only to fill the inventory?

Reply:
Done 

![image](https://user-images.githubusercontent.com/67913809/155559214-46cf0c0b-c0d1-4f35-b8d0-f54e9bb9ded3.png)